### ISSUE TITLE
CDB: Better onboarding experience to Cockroach Cloud

### ISSUE THREAD

**Issue by matthewmueller** [prisma/prisma#10494]
As a new user, you'd expect to be able to do:

1. Go to https://www.cockroachlabs.com/
2. Click "Start Instantly"
3. Create a "CockroachDB Serverless" "Start Instantly" button
4. Signup and provision a database
5. Get my connection string 
6. Add it to the .env file of a new prisma project
7. Run npx prisma db push 

Unfortunately, this leads to:

```
Encounter: db error: FATAL: codeParamsRoutingFailed: missing cluster name in connection string
```

There's a bit of information on how to set it up in [this comment](https://github.com/prisma/prisma/issues/9978#issuecomment-963387850). 

We should work with CockroachDB to improve this workflow.

**Comment by janpio** [comment]
Initial investigation this could be fixed. Issues: https://github.com/prisma/prisma/issues/10805 + https://github.com/prisma/quaint/issues/338

**Comment by janpio** [comment]
In recent `dev` versions of Prisma CLI the connection string com CockroachDB Cloud Serverless can just be copy pasted into Prisma and works. No additional work is need any more. This will be released with 3.8.0.
